src/coreclr/tools/aot/ILCompiler/ILCompiler.csproj

```diff
@@ -1,18 +1,28 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputPath>$(RuntimeBinDir)ilc/</OutputPath>
-    <RuntimeIdentifier>$(PackageRID)</RuntimeIdentifier>
+    <RuntimeIdentifier>$(OutputRID)</RuntimeIdentifier>
+
+    <!-- Mobile platforms don't have apphost and we don't need it. However, restore requires
```

Comment:
We are not shipping naot that runs on phones. For example, https://nuget.info/packages/runtime.ios-arm64.Microsoft.DotNet.ILCompiler/8.0.0-preview.6.23329.7 contains target libraries, but no compiler. The compiler is expected to come from the host package (e.g. osx-arm64).

Would it be better to stop building ILCompiler for mobile instead of these workarounds?

Reply:
Good point. I will give it a try. :)